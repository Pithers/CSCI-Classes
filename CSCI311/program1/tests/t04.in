table table-01 1 Julia
table table-02 1 Julia
table table-03 1 Julia
table table-04 99 Julia
party 1 Larry 60
party 1 Mike 75
party 66 Cindy 90
party 5 Lenny 60
table table-05 100 Julia
party 98 Jim 1
end
