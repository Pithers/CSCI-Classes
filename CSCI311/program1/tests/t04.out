Larry seated at table-01
Mike seated at table-02
Cindy seated at table-04
Lenny seated at table-05
Larry finished at table-01
Lenny finished at table-05
Jim seated at table-05
Jim finished at table-05
Mike finished at table-02
Cindy finished at table-04
Julia served 171
